packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -120,9 +124,12 @@ class FlexibleSpaceBar extends StatefulWidget {
   /// inset from the bottom-left and it is specified along with
   /// [centerTitle] false.
   ///
-  /// By default the value of this property is
-  /// `EdgeInsetsDirectional.only(start: 72, bottom: 16)` if the title is
-  /// not centered, `EdgeInsetsDirectional.only(start: 0, bottom: 16)` otherwise.
+  /// If [centerTitle] is true, then the title is centered within the then
+  /// flexible space bar with a bottom padding of 16.0 pixels.
+  ///
+  /// If [centerTitle] is false and [FlexibleSpaceBarSettings.hasLeading] is true,
+  /// then the title is aligned to the start of the flexible space bar with a start
+  /// padding of 72.0 pixels and a bottom padding of 16.0 pixels.
```

Comment:
Is it possible the leading widget consumes more than 72 pixels? If so, how is that scenario resolved?

Reply:
When `titlePadding` parameter is not null, this will use the provided value. Updated the docs